src/components/AssetDownload/index.tsx

```diff
@@ -52,34 +55,13 @@ const AssetDownload: React.FC<IProps> = ({
         )}
       </Box>
       <Flex gap={5} mt={4}>
-        {downloadUrl && (
-          <ButtonLink
-            to={downloadUrl}
-            onClick={() => {
-              trackCustomEvent({
-                eventCategory: "asset download button",
-                eventAction: "click",
-                eventName: title,
-              })
-            }}
-          >
-            <Translation id="page-assets-download-download" />
-            <>&nbsp;(PNG)</>
-          </ButtonLink>
-        )}
+        <ButtonLink href={image.src} onClick={matomoHandler} target="_blank">
```

Comment:
In Gatsby repo, we're defining `const baseUrl = 'https://ethereum.org'`, which forces this to an "external" link... This breaks in development for this migration because these paths don't actually live at that domain yet...
I tried switching to simply using the relative path, which works, but it results in the image opening in the same tab, which is why I added the `target="_blank"`. The `ButtonLink` component does not accept `isExternal` boolean—open to ideas on a clean way to handle this to avoid using `target` attribute.

Reply:
Good.

> The ButtonLink component does not accept isExternal boolean—open to ideas on a clean way to handle this to avoid using target attribute.

Agree. An `isExternal` prop would be beneficial. I'd also like to remove the `isExternal` logic from the Link component in favor of the prop but that would require a big refactor in the codebase.